app/lib/analyzer/handlers.dart

```diff
@@ -16,15 +16,14 @@ import 'models.dart';
 
 /// Handlers for the analyzer service.
 Future<shelf.Response> analyzerServiceHandler(shelf.Request request) async {
-  final path = request.requestedUri.path;
   final handler = {
     '/debug': debugHandler,
-  }[path];
+  }[request.requestedUri.path];
 
   if (handler != null) {
     return handler(request);
-  } else if (path.startsWith('/packages/')) {
-    return packageHandler(request);
+  } else if (request.url.pathSegments.first == 'packages') {
+    return packageHandler(request.change(path: 'packages'));
```

Comment:
This seems to clear the `package/version` part from the path, is that the intention?

Reply:
Yup! Let's the handler live it's in sub-path world without caring about the global path mapping

Reply:
If we won't use the `path`, why change it?

Reply:
It removes it from the URL – so the handler doesn't have to think about it.

See the code below that changed. You no longer have to ignore the first package segment – it's just not there.

Reply:
I see that below, but I find it confusing that we are changing the path here, as if there is a reason or use for it.